website/docs/plugin/framework/migrating/attributes-blocks/attribute-schema.mdx

```diff
@@ -0,0 +1,129 @@
+
+The following code shows a basic implementation of attribute schema for a provider in SDKv2. For clarity, the body of
+the attribute's schema is omitted (denoted by `...`).
+
+```go
+func ProviderExample() *schema.Provider {
+	return &schema.Provider{
+        Schema: map[string]*schema.Schema{
+	...
+},
+```
+
+In SDKv2, resource and data source attributes are defined the same way on their respective types.
+
+```go
+func resourceExample() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			...
+		},
+```
+## Framework
+
+In the Framework, you define attributes by setting the `Attributes` field on your provider, resource, or data type's
+schema, as returned by the `GetSchema` function. The `tfsdk.Schema` type returned by `GetSchema` includes an
+`Attributes` field that maps each attribute name (string) to the attribute's `tfsdk.Attribute` struct.
+
+The following code shows how to define an attribute for a resource with the Framework. For clarity, the body of the
+`tfsdk.Attribute` struct is omitted (denoted by `...`).
+
+```go
+func (d *resourceTypeExample) GetSchema(context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		Attributes: map[string]tfsdk.Attribute{
+			"example": {
+			    ...
+```
+
+The Framework defines the `tfsdk.Attribute` struct as follows.
+
+```go
+type Attribute struct {
+	Type                attr.Type
+	Attributes          NestedAttributes
+	Description         string
+	MarkdownDescription string
+	Required            bool
+	Optional            bool
+	Computed            bool
+	Sensitive           bool
+	DeprecationMessage  string
+	Validators          []AttributeValidator
+	PlanModifiers       AttributePlanModifiers
+}
+```
+## Migration Notes
+Remember the following differences between SDKv2 and the Framework when completing the migration.
+
+- In SDKv2, attributes are defined by a map from attribute names to `schema.Schema` structs in the `Schema` field of
+your resource's schema. In the Framework, attributes are defined by a map from attribute names to `tfsdk.Attribute`
+structs in your resource's schema, which is returned by the `GetSchema` function.
+- There are several differences between the implementation of attributes in SDKv2 and the Framework. Refer to the other
+pages in the Attributes & Blocks section of this migration guide for more details.
+
+## Example
+
+The following examples show how to migrate portions of the
+[http](https://github.com/hashicorp/terraform-provider-http) provider.
+
+For clarity, some parts of the migration are omitted (denoted by `...`). For a complete example, clone the
+`terraform-provider-http` repository and compare the `data_source.go` file in
+[v2.2.0](https://github.com/hashicorp/terraform-provider-http/blob/v2.2.0/internal/provider/data_source.go)
+and the `data_source_http.go` file in
+[v3.0.1](https://github.com/hashicorp/terraform-provider-http/blob/v3.0.1/internal/provider/data_source_http.go).
+
+
+**SDKv2**
```

Comment:
Yeah you know what? @bendbennett I am looking at the rendered content and the bolding doesn't work the way I thought it did haha. Can you please change all of these in the ## Example section so that they are one Heading level below whatever the example heading is? Often I believe that will be H3, but it may be lower, depending on what the Example heading is set at. 

<img width="960" alt="Screen Shot 2022-08-31 at 12 14 04 PM" src="https://user-images.githubusercontent.com/83350965/187727699-1260e4d7-1a48-45fd-937b-cef7bddc7cbb.png">

Reply:
Fixed